packages/fuels/tests/contracts.rs

```diff
@@ -1332,120 +1331,6 @@ async fn low_level_call() -> Result<()> {
     Ok(())
 }
 
-#[cfg(any(not(feature = "fuel-core-lib"), feature = "rocksdb"))]
```

Comment:
This test shouldn't be completely removed since we still support rocksdb as a database type but only when we're using the `fuel-core` as a binary.